app/entry.client.tsx

```diff
@@ -1,18 +1,57 @@
-/**
- * By default, Remix will handle hydrating your app on the client for you.
- * You are free to delete this file if you'd like to, but if you ever want it revealed again, you can run `npx remix reveal` ✨
- * For more information, see https://remix.run/docs/en/main/file-conventions/entry.client
- */
-
-import { RemixBrowser } from '@remix-run/react';
-import { startTransition, StrictMode } from 'react';
-import { hydrateRoot } from 'react-dom/client';
-
-startTransition(() => {
-  hydrateRoot(
-    document,
-    <StrictMode>
-      <RemixBrowser />
-    </StrictMode>
-  );
-});
+/**
+ * By default, Remix will handle hydrating your app on the client for you.
+ * You are free to delete this file if you'd like to, but if you ever want it revealed again, you can run `npx remix reveal` ✨
+ * For more information, see https://remix.run/docs/en/main/file-conventions/entry.client
+ */
+
+import { RemixBrowser } from '@remix-run/react';
+import { startTransition, StrictMode } from 'react';
+import { hydrateRoot } from 'react-dom/client';
+
+startTransition(() => {
+  hydrateRoot(
+    document,
+    <StrictMode>
+      <RemixBrowser />
+    </StrictMode>
+  );
+});
+
+if ('serviceWorker' in navigator) {

```

Comment:
isso aqui vc pegou referencia de algum lugar? @ruvalter ?
to perguntando pra saber se vc colocou esse codigo aqui vendo de outro lugar ou foi da sua cabeça mesmo
to na duvida se aqui nao pode dar problema com o hydrate...enfim....nao sei  dizer...mas se foi algo que foi recomendado, aí beleza

Reply:
this line was added by the react-pwa lib.